pkg/cmd/server/api/v1/types.go

```diff
@@ -626,3 +637,16 @@ type PodManifestConfig struct {
 	// The interval needs to be a positive value
 	FileCheckIntervalSeconds int64 `json:"fileCheckIntervalSeconds"`
 }
+
+type ExtensionFilesConfig struct {
+	// SubContext is the path under /<context>/extensions/ to serve files from Location
+	SubContext string `json:"subcontext"`
```

Comment:
any reason we can't just call this `name` and say extensions are served under `/extensions/<name>`?